packages/v4/src/content/design-guidelines/usage-and-behavior/filters/filters.md

```diff
@@ -12,210 +12,315 @@ related: [
+
+The following elements make up an attribute-value (textbox) filter.
+1. **Attribute selector:** A select list that allows the user to select the attribute that they want to filter against.
+2. **Value selector:** A value selector that is used to pass a value to the filter. The value selector can be any data input component. Common components used for value selection include:
+  
+    a. A **text entry field** for entering a value from the keyboard. Use when you will enter the value as unbounded data (e.g. a name). When the text entry field is a search input, the value will appear as a chip automatically, and the search will start being validated. When the text entry field is an input group, the value will appear as a chip and the search will be validated when the user presses the Enter key on their keyboard or the search button. 
+   
+    b. A **single select** for selecting a value from a predefined list. This will work the same as a stand alone single select, however since any selection in an attribute filter will result in a chip, there is no need to have an “All” option as part of your menu, as users can reset their filters using the chips.
+    
+    c. A **checkbox select** for selecting multiple values from a predefined list. Multiple values will be combined using a logical OR operation. Items that meet multiple values will be shown higher in the results list.
+    
+    d. A **date picker** for selecting a date range. Use when the value you want to filter on is expressed as a date and/or time stamp.
 
 #### Behavior
-A filter query is triggered whenever a new value is set by the user. The following are triggers applicable to the variations of the attribute-value filter shown above.
+To use an attribute-filter, the user must first select an _attribute_ to filter on using the left toggle, followed by choosing the specific _values_ to filter on using the right toggle. Whatever is selected in the attribute toggle will determine what is shown in the value toggle.
+
+The value toggles behave as described in the [variations for one attribute](#variations-for-one-attribute) section. The only difference for multiple attributes is that as users make filter selections using the attribute filter, each selection will **always** show up as a chip underneath the filters.
+
+When using the attribute filter, a filter query is triggered whenever the user sets a new value. The following are triggers applicable to the variations of the attribute-value filter shown above.
 
 | Value selector | Trigger |
-| ------- | ------ |
-| Search field | The user presses the Enter key or clicks the Search button. |
+| ------------------ | ------ |
+| Search field | The user starts typing (for search input), presses the Enter key, or clicks the Search button (for input groups). |
 | Select lists | The user selects or deselects an item in the list. |
 | Date picker | The user selects a new date and/or time. |
 
```

Comment:
@mmenestr The first column of the table isn't rendering fully and is cutting off the "value selector" word. You might have to edit the width of it!